proposals/csharp-9.0/native-integers.md

```diff
@@ -77,6 +77,8 @@ The tables below cover the conversions between special types.
 | `decimal` | `nuint` | ExplicitNumeric | `ulong decimal.op_Explicit(decimal) conv.u` / `... conv.ovf.u.un`  |
 | `IntPtr` | `nuint` | None | |
 | `UIntPtr` | `nuint` | Identity | |
+|Enumeration|`nint`|ExplicitEnumeration||
+|Enumeration|`nuint`|ExplicitEnumeration||
```

Comment:
I'm unsure of the IL, so left them empty for now until someone confirms the correct IL.

Reply:
@jcouv @cston, the table is still missing the IL instructions in the last column

Reply:
The IL depends on the underlying type of the `enum` so it seems fine to leave the column blank.